inspirehep/modules/workflows/workflows/edit_article.py

```diff
@@ -50,6 +62,7 @@ class EditArticle(object):
 
     workflow = (
         [
+            update_wf_payload_from_db,
```

Comment:
This should not be needed, as when creating the workflow we are putting in it the record from the db already: https://github.com/inspirehep/inspire-next/blob/master/inspirehep/modules/workflows/views.py#L676

Reply:
aren't you just the wait time for the record to round-trip through legacy?